src/app/code/community/Divante/VueStorefrontIndexer/Model/Index/Mapping/Eav/Abstract.php

```diff
@@ -14,6 +14,20 @@
  */
 abstract class Divante_VueStorefrontIndexer_Model_Index_Mapping_Eav_Abstract
 {
+    /**
+     * @var array
+     */
+    protected $longProperties = [
+        'category_ids'
```

Comment:
missing 'level' property

Reply:
Added level property.
But all properties in getAttributeMapping() are captured in here. Everything additional should be done beside of this PR.